### ISSUE TITLE
Crash with fatal error (0.3.2)

### ISSUE THREAD

**Issue by caffeine-mgn** [allusive-dev/picom-allusive#8]
Kernel: 6.2
OS: Majaro
CPU: amd 5950X
GPU: Nvidia 3090
DE: Awesome 4.3
```
[ 15.10.2023 13:59:59.572 x_new_id FATAL ERROR ] We seems to have run of XIDs. This is either a bug in the X server, or a resource leakage in the compositor. Please open an issue about this problem. The compositor will die.
zsh: IOT instruction (core dumped)  picom
```

Also noticed friezes some freeze when after 2-3 hour and i switch screen work space.

[picom.conf.txt](https://github.com/allusive-dev/picom-allusive/files/12909379/picom.conf.txt)
My config

**Comment by allusive-dev** [comment]
Like I said in the other issue I cannot do much at the moment but first please try testing picom with the following conditions.

Remove everything to do with transistions as I Sm pretty sure this fork dosent implement that.

Second try just running the sample configuration and let me know if there are any issues or the same issues for either of these.

Thanks

**Comment by allusive-dev** [comment]
@caffeine-mgn This issue should hopefully be resolved as of `v1.2` or later.

Let me know if that is not the case.
